e2e/tests-dfx/start.bash

```diff
@@ -43,7 +42,7 @@ teardown() {
 }
 
 @test "stop and start with other options does not disrupt projects" {
-  [[ "$USE_POCKETIC" ]] && skip "skipped for pocketic: artificial delay"
+  [[ "$USE_POCKETIC" ]] && skip "skipped for pocketic: clean required"
```

Comment:
Why is clean an issue?

Reply:
This is a test for #3777, which doesn't (yet) apply to PocketIC. This can be addressed in a future PR, I'm just updating the comments to indicate what's the remaining required feature.